static/msredirect.html

```diff
@@ -0,0 +1,11 @@
+<html>
```

Comment:
I'm wondering if it's best to include the doctype. According to W3C it is required but I don't think omitting it actually causes much harm for this case.